docs/quickstarts/packet.md

```diff
@@ -0,0 +1,225 @@
+
+You can configure the components as per the requirements.
+
+Create a variables file `lokocfg.vars` to define the variables present in the configuration file.
+
+```console
+#lokocfg.vars
+
+packet_project_id = "PACKET_PROJECT_ID"
+ssh_public_keys = ["public-ssh-key-1", "public-ssh-key-2", ...]
+
+state_s3_bucket = "name-of-the-s3-bucket-to-store-the-cluster-state"
+lock_dynamodb_table = "name-of-the-dynamodb-table-for-state-locking"
+
+dns_zone = "dns-zone-name"
+route53_zone_id = "zone-id-of-the-dns-zone"
+
+management_cidrs = "public-ip-address-cidr-to-access-the-cluster"
+node_private_cidr = "private-subnet-assigned-by-packet-to-the-project"
+
+cert_manager_email = "email-address-used-for-cert-manager-component"
+grafana_admin_password = "password-for-grafana"
+```
+
+For advanced cluster configurations and more information refer to the [Packet configuration
+guide.](../configuration-guides/platforms/packet.md)
+
+### Step 5: Create Lokomotive cluster
+
+Run the following command to create the cluster:
+
+```console
+lokoctl cluster install
+```
+
+Once the command finishes, your Lokomotive cluster details are stored in the path you've specified
+under `asset_dir`.
+
+## Verification
+
+A successful installation results in the output:
+
+```console
+module.packet-mycluster.null_resource.bootkube-start: Still creating... [4m10s elapsed]
+module.packet-mycluster.null_resource.bootkube-start: Still creating... [4m20s elapsed]
+module.packet-mycluster.null_resource.bootkube-start: Creation complete after 4m25s [id=1122239320434737682]
+
+Apply complete! Resources: 74 added, 0 changed, 0 destroyed.
+
+Your configurations are stored in /home/imran/lokoctl-assets/mycluster
+
+Now checking health and readiness of the cluster nodes ...
+
+Node                                          Ready    Reason          Message
+
+mycluster-controller-0                        True     KubeletReady    kubelet is posting ready status
+mycluster-controller-1                        True     KubeletReady    kubelet is posting ready status
+mycluster-controller-2                        True     KubeletReady    kubelet is posting ready status
+mycluster-worker-pool-1-worker-0              True     KubeletReady    kubelet is posting ready status
+mycluster-worker-pool-1-worker-1              True     KubeletReady    kubelet is posting ready status
+mycluster-worker-pool-1-worker-2              True     KubeletReady    kubelet is posting ready status
+
+Success - cluster is healthy and nodes are ready!
+```
+
+Use the generated `kubeconfig` file to access the Kubernetes cluster and list nodes.
+
+```console
+export KUBECONFIG=./lokomotive-assets/auth/kubeconfig
+kubectl get nodes
+```
+
+## Cleanup
+
+To destroy the Lokomotive cluster, execute the following command:
+
+```console
+lokoctl cluster destroy --confirm
+```
+
+You can safely delete the working directory created for this quickstart guide if you no longer
+require it.
+
+## Troubleshooting
```

Comment:
I think this section should also include mention of insufficient availability of node types on Packet. That sometimes you may get an error, that there is no machines of type X available in region and then it's best to change the region or machine type used and refer to availability page of Packet (if one exists, I know it's available from the API).

We could also mention, that node types with spare disks should be used, otherwise installing prometheus operator component will fail.